tests/integration/telemetry/stats/prometheus/customizemetrics/customize_metrics_test.go

```diff
@@ -212,6 +232,26 @@ func setupEnvoyFilter(ctx resource.Context) error {
 		return err
 	}
 
+	// enable custom tag in the stats
+	bootstrapPatch := `
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: bootstrap-tag
+spec:
+  configPatches:
```

Comment:
Can we also add a test for `stats_matcher` ? We do have specific annotations but would it be better to have BDS cover everything ?

Reply:
We could use EnvoyFilter for stats inclusions. I just didn't want to create a brand-new integration test since it requires a new echo deployment. Do you want to just check envoy stats for presence of some extra tag with the right bootstrap patch?

Reply:
Yes that should suffice. Someone is going to use it that way.

Reply:
Let's do that in a follow up. This customize metric test is rather hard to set up right (and it took awhile to debug it), so let's not complicate it further.

Reply:
Lgtm.